app/helpers/url_helper.rb

```diff
@@ -0,0 +1,19 @@
+module UrlHelper
```

Comment:
same for this, comment would be nice

Reply:
Sure, for both. You feeling okay-ish about this PR in general? Shall I proceed? Or should we try something else? I did think briefly about attempting to make the prev/next toggles take on the context of the bento search click. But I wasn't sure that would even be desired or expected and given that I think it'd be a fair amount of work to implement (possibly?)

Reply:
Yes, I think this approach makes sense and it is the upstream sol'n just not merged yet. Let's go with this, once it is merged, we will take out our overrides 👍  thanks!

Reply:
it makes sense to not show a the prev/next when the user not coming from a search context within BL, i think it is OK to hide them when coming from bento

Reply:
cool! Thanks Banu

Reply:
> same for this, comment would be nice

We don't add a comment for the other "blacklight helpers" that we override - facets and layouts. I think we're ok without a comment for this one. ?